pokemon_battle_arena/src/db/pokemon_token.rs

```diff
@@ -67,59 +67,45 @@ impl PokemonToken {
             mega_evolution: model.get_mega(),
         }
     }
-
     pub fn is_asleep(&self) -> bool {
-        self.non_volatile_status.0 == enums::Non_Volatile::Sleep
+        self.non_volatile_status.0 == enums::NonVolatile::Sleep
     }
-
     pub fn get_moves(&self, dex: movedex::Movedex) -> movedex::Movedex {
         dex.for_token(self.get_level(), self.pokedex_id)
     }
-
     pub fn get_id(&self) -> usize {
         self.pokedex_id
     }
-
     pub fn get_name(&self) -> String {
         self.clone().name
     }
```

Comment:
Think about what you are doing here. You are returning an owned string (which is already questionable!). In order to do that you clone the *entire* `self`. Not just the string, but the whole object. Writing `self.name.clone()` instead is already way better. And again: you probably don't need to return an owned string anyway.